docs/aws/karpenter.cloudformation.yaml

```diff
@@ -0,0 +1,88 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/awslabs/karpenter
+Parameters:
+  OpenIDConnectIdentityProvider:
+    Type: String
+    Description: "Example oidc.eks.us-west-2.amazonaws.com/id/1234567890. "
+Mappings:
+  ServicePrincipals:
+    aws-cn:
+      ec2: ec2.amazonaws.com.cn
+    aws-us-gov:
+      ec2: ec2.amazonaws.com
+    aws:
+      ec2: ec2.amazonaws.com
+Resources:
+  KarpenterControllerRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: KarpenterControllerRole
+      Path: /
+      AssumeRolePolicyDocument: !Sub |
+        {
+          "Version": "2012-10-17",
+          "Statement": [{
+            "Effect": "Allow",
+            "Principal": {
+              "Federated": "arn:aws:iam::${AWS::AccountId}:oidc-provider/${OpenIDConnectIdentityProvider}"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+              "StringEquals": {
+                "${OpenIDConnectIdentityProvider}:aud": "sts.amazonaws.com",
```

Comment:
You might need to use `AWS::URLSuffix` here too?

Reply:
+1